pmlc/tools/pmlc-opt/BUILD

```diff
@@ -9,13 +9,7 @@ plaidml_cc_binary(
     copts = COPTS,
     visibility = ["//visibility:public"],
     deps = [
-        "//pmlc/dialect/eltwise",
-        "//pmlc/dialect/stripe",
-        "//pmlc/dialect/tile",
-        "@mlir//:EDSC",
-        "@mlir//:OptMain",
-        "@mlir//:Parser",
-        "@mlir//:TestTransforms",
-        "@mlir//:Transforms",
+        "//pmlc/conversion/stripe_to_affine",
+        "@mlir//:MlirOptLib",
     ],
```

Comment:
Even though `//pmlc/conversion/stripe_to_affine` also depends on other things (e.g. `//pmlc/dialect/eltwise`) it's still better to explicitly list them here because `pmlc-opt` independently depends on these things.